pointercrate-demonlist/sql/paginate_players_by_id.sql

```diff
@@ -1,11 +1,13 @@
-SELECT id, name::TEXT, banned, nation::TEXT, iso_country_code::TEXT, players.score
+SELECT id, players.name::TEXT, banned, nationalities.nation::TEXT, iso_country_code::TEXT, subdivision::TEXT AS iso_code, subdivisions.name AS subdivision_name, players.score
 FROM players
 LEFT OUTER JOIN nationalities ON nationality = iso_country_code
+LEFT JOIN subdivisions ON iso_code = subdivision AND subdivisions.nation = nationality
```

Comment:
Phew, it's been a hot minute since I had to deal with non-trivial SQL

This join also has to be a `LEFT OUTER JOIN`, as otherwise all players that do not have a subdivision set will be discarded from the result set.